### DISCUSSION TITLE
Abandoned?

### DISCUSSION THREAD

**Discussion by jezmck** [vercel/hyper#7770] [General]
There's been no update for 6 months, just wondering whether @LabhanshAgrawal is okay?

**Comment by LabhanshAgrawal** [comment]
Hey, yeah, I had some things going on which had me occupied. Will be back in action soon. Thanks for checking in.

**Reply by tony19** [reply]
Friendly bump 😊 Just curious if you're still planning on new features, as I haven't seen any official releases since 01/2023.

**Reply by mchisolm0** [reply]
I’m also very interested. 😊
